identity-service/src/routes/healthCheck.js

```diff
@@ -92,7 +92,7 @@ module.exports = function (app) {
       if (block && block.transactions.length) {
         for (const tx of block.transactions) {
           // If transaction is from audius account, determine success or fail status
-          if (RELAY_HEALTH_ACCOUNT === tx.from) {
+          if (RELAY_HEALTH_ACCOUNTS.includes(tx.from)) {
```

Comment:
just because it's perf sensitive, and already in looks, you could also instantiate a set of wallets on server init

RELAY_WALLETS = new Set(config.get('relayerWallets').map(wallet => wallet.publicKey))

and then do RELAY_WALLETS.has(tx.from)